docs/api/users-delete-user-by-id.md

```diff
@@ -0,0 +1,54 @@
+
```

Comment:
```suggestion
```
Please remove the blank lines before the title.
The title should be the first line of the file, unless there is metadata.

Reply:
Done. Thanks.